avocado/utils/configure_network.py

```diff
@@ -35,111 +35,176 @@ class NWException(Exception):
     """
 
 
-def set_ip(ipaddr, netmask, interface, interface_type=None):
+class Network_interface:
```

Comment:
Class name should follow PEP8: `NetworkInterface`

Reply:
sure